lib/std/fs.zig

```diff
@@ -1731,7 +1771,7 @@ pub const Dir = struct {
             null,
             0,
```

Comment:
I might be off here, but my hunch tells me you want `FILE_ATTRIBUTE_NORMAL` if creating a dir, `0` otherwise.

Reply:
Oh yep, Windows docs confirm it [ZwCreateFile](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nf-wdm-zwcreatefile):

> [in] FileAttributes

> Specifies one or more FILE_ATTRIBUTE_XXX flags, which represent the file attributes to set if you create or overwrite a file. The caller usually specifies FILE_ATTRIBUTE_NORMAL, which sets the default attributes. For a list of valid FILE_ATTRIBUTE_XXX flags, see the [CreateFile](https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea) routine in the Microsoft Windows SDK documentation. If no file is created or overwritten, FileAttributes is ignored.